docs/tutorials/substreams/substreams-app-example.experimental.md

```diff
@@ -0,0 +1,215 @@
+---
+title: 'Substreams application example'
+
+order: -9999
+outline: [0,4]
+---
+
+# Substreams Application Example
```

Comment:
Should need some explanation about Substreams sink, because the term "Substreams application" is not used in the streaming fast document, the user may get confused when looking into streaming fast document.

Reply:
Updated to `# Substreams Sink Application Example`
And added `Any application that is being used to consume `Substreams data` will be referred as `Substreams Sink`.`